DxDiagram improve Expr descriptions (#620)
DevExpress/devextreme-documentation: api-reference/10 UI Widgets/dxDiagram/1 Configuration/edges/fromPointIndexExpr.md

The current edge's data object.

---

> The built-in shape's connection points are numbered clockwise starting with the leftmost point on the top border.

Reviewer: ```suggestion
The built-in shape's connection points are numbered clockwise from the leftmost point on the top border.
```
